packages/@glimmer/runtime/lib/dom/normalize.ts

```diff
@@ -40,7 +40,7 @@ export function shouldCoerce(
 }
 
 export function isEmpty(value: unknown): boolean {
-  return value === null || value === undefined || typeof (value as Dict).toString !== 'function';
+  return value === undefined || value === null || typeof (value as Dict).toString !== 'function';
```

Comment:
empty values likely undefined, and less likely null

Reply:
Isn't it better to just do `value == null`?

Reply:
I'm not trying to change logic of this function, just ordering code paths to get more "matches" first (in my understanding in any web app `undefined` could be found more often than `null`